ui/static/ui/css/mit-lore.css

```diff
@@ -251,6 +251,11 @@ a {
     padding: 0;
 }
 
+.tile-meta {
+	font-size:small;
+	color:#929292;
+}
+
```

Comment:
.tile-meta is defined twice in this file.  Remove this instance and modify the other on line 327.